tests/tests/test-precompile/test-precompile-wormhole.ts

```diff
@@ -209,6 +210,27 @@ describeDevMoonbeam(`Test local Wormhole`, (context) => {
       .signAndSend(alith);
     await context.createBlock();
 
+    // create payload
+    const versionedMultiLocation = {
+      v2: {
+        parents: 1,
+        interior: {
+          X2: [
+            { Parachain: 1001 },
+            { AccountId32: { id: "0x0000000000000000000000000000000000000000000000000000000000000000"}}
+          ]
+        }
+      }
+    };
+
+    const destination = context.polkadotApi.registry.createType("VersionedMultiLocation", versionedMultiLocation);
+
+    const userAction = new XcmRoutingUserAction({ destination });
+    const versionedUserAction = new VersionedUserAction({ V1: userAction });
+    console.log("Versioned User Action JSON:", versionedUserAction.toJSON());
+    console.log("Versioned User Action SCALE:", versionedUserAction.toHex());
+    const payload = versionedUserAction.toHex();
```

Comment:
This payload ends up being shorter than it should be by exactly 12 bytes. Adding these as `0`s manually allows the decoding to succeed. Perhaps the `AccountId32` is somehow being confused with `AccountId20`?

Reply:
Di you try import "@moonbeam-network/api-augment"? I think if u use the default polkadotjs types, it sets AccountID32 junction to whatever your accountId is (in this case 20 bytes).

Reply:
Thanks, good idea but didn't seem to help. It occurs to me that, at this point at least, it doesn't matter what account I use, so changing to `AccountKey20` makes the SCALE decoding happy.

Reply:
Could this test be rewritten as a para test (e.g. `describeParachain`)? I should be able to verify not only the entire precompile interaction, but also XCM receive on the destination chain that way.

Reply:
propbably yes, moonwall makes it very easy to write those tests. AFAIK Moonbeam should already be prepared for that @timbrinded